amf-api-contract/shared/src/main/scala/amf/apicontract/internal/spec/common/emitter/ServerEmitters.scala

```diff
@@ -280,7 +280,7 @@ case class OasServerVariableEmitter(variable: Parameter, ordering: SpecOrdering)
     val shape  = variable.schema
     val fs     = shape.fields
 
-    fs.entry(ShapeModel.Description).map(f => result += ValueEmitter("description", f))
+    variable.fields.entry(ShapeModel.Description).map(f => result += ValueEmitter("description", f))
```

Comment:
This should be ParameterModel.Description

Reply:
Done